src/borg/locking.py

```diff
@@ -191,6 +191,8 @@ def kill_stale_lock(self):
             names = os.listdir(self.path)
         except FileNotFoundError:  # another process did our job in the meantime.
             pass
```

Comment:
hmm, thinking about 193:

guess `pass` is a bit pointless (as it runs into the `os.rmdir(self.path)` then, for a non-existing path).

so we could also just `return False` here.